python/mcap/mcap/writer.py

```diff
@@ -427,9 +434,13 @@ def __finalize_chunk(self):
 
         chunk_data = self.__chunk_builder.end()
         if self.__compression == CompressionType.LZ4:
+            if lz4 is None:
+                raise RuntimeError("lz4 is not installed")
```

Comment:
This is a configuration error, the error should be raised at configuration-time, not here when finalizing the chunk.